### ISSUE TITLE
2025-04-22, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#148]
# Today's Arxiv Papers

(1) [VisuLogic: A Benchmark for Evaluating Visual Reasoning in Multi-modal Large Language Models](http://arxiv.org/abs/2504.15279v1), 2025-04-21

(2) [Seeing from Another Perspective: Evaluating Multi-View Understanding in MLLMs](http://arxiv.org/abs/2504.15280v1), 2025-04-21

(3) [Stop Summation: Min-Form Credit Assignment Is All Process Reward Model Needs for Reasoning](http://arxiv.org/abs/2504.15275v1), 2025-04-21

(4) [Interpretable Locomotion Prediction in Construction Using a Memory-Driven LLM Agent With Chain-of-Thought Reasoning](http://arxiv.org/abs/2504.15263v1), 2025-04-21

(5) [CRUST-Bench: A Comprehensive Benchmark for C-to-safe-Rust Transpilation](http://arxiv.org/abs/2504.15254v1), 2025-04-21

(6) [Evaluating Judges as Evaluators: The JETTS Benchmark of LLM-as-Judges as Test-Time Scaling Evaluators](http://arxiv.org/abs/2504.15253v1), 2025-04-21

(7) [MR. Guard: Multilingual Reasoning Guardrail using Curriculum Learning](http://arxiv.org/abs/2504.15241v1), 2025-04-21

(8) [A Self-Improving Coding Agent](http://arxiv.org/abs/2504.15228v1), 2025-04-21

(9) [EvalAgent: Discovering Implicit Evaluation Criteria from the Web](http://arxiv.org/abs/2504.15219v1), 2025-04-21

(10) [Integrating Symbolic Execution into the Fine-Tuning of Code-Generating LLMs](http://arxiv.org/abs/2504.15210v1), 2025-04-21

(11) [Compute-Optimal LLMs Provably Generalize Better With Scale](http://arxiv.org/abs/2504.15208v1), 2025-04-21

(12) [Support Evaluation for the TREC 2024 RAG Track: Comparing Human versus LLM Judges](http://arxiv.org/abs/2504.15205v1), 2025-04-21

(13) [Zero-Shot, But at What Cost? Unveiling the Hidden Overhead of MILS's LLM-CLIP Framework for Image Captioning](http://arxiv.org/abs/2504.15199v1), 2025-04-21

(14) [Synergistic Weak-Strong Collaboration by Aligning Preferences](http://arxiv.org/abs/2504.15188v1), 2025-04-21

(15) [DSPO: Direct Semantic Preference Optimization for Real-World Image Super-Resolution](http://arxiv.org/abs/2504.15176v1), 2025-04-21

(16) [The Synthetic Imputation Approach: Generating Optimal Synthetic Texts For Underrepresented Categories In Supervised Classification Tasks](http://arxiv.org/abs/2504.15160v1), 2025-04-21

(17) [KGMEL: Knowledge Graph-Enhanced Multimodal Entity Linking](http://dx.doi.org/10.1145/3726302.3730217), 2025-04-21

(18) [EasyEdit2: An Easy-to-use Steering Framework for Editing Large Language Models](http://arxiv.org/abs/2504.15133v1), 2025-04-21

(19) [Kuwain 1.5B: An Arabic SLM via Language Injection](http://arxiv.org/abs/2504.15120v1), 2025-04-21

(20) [Rethinking the Potential of Multimodality in Collaborative Problem Solving Diagnosis with Large Language Models](http://arxiv.org/abs/2504.15093v1), 2025-04-21

(21) [Empowering AI to Generate Better AI Code: Guided Generation of Deep Learning Projects with LLMs](http://arxiv.org/abs/2504.15080v1), 2025-04-21

(22) [Think2SQL: Reinforce LLM Reasoning Capabilities for Text2SQL](http://arxiv.org/abs/2504.15077v1), 2025-04-21

(23) [The Great Nugget Recall: Automating Fact Extraction and RAG Evaluation with Large Language Models](http://arxiv.org/abs/2504.15068v1), 2025-04-21

(24) [Testing LLMs' Capabilities in Annotating Translations Based on an Error Typology Designed for LSP Translation: First Experiments with ChatGPT](http://arxiv.org/abs/2504.15052v1), 2025-04-21

(25) [ScanEdit: Hierarchically-Guided Functional 3D Scan Editing](http://arxiv.org/abs/2504.15049v1), 2025-04-21

(26) [RainbowPlus: Enhancing Adversarial Prompt Generation via Evolutionary Quality-Diversity Search](http://arxiv.org/abs/2504.15047v1), 2025-04-21

(27) [A Call for New Recipes to Enhance Spatial Reasoning in MLLMs](http://arxiv.org/abs/2504.15037v1), 2025-04-21

(28) [DyST-XL: Dynamic Layout Planning and Content Control for Compositional Text-to-Video Generation](http://arxiv.org/abs/2504.15032v1), 2025-04-21

(29) [DistilQwen2.5: Industrial Practices of Training Distilled Open Lightweight Language Models](http://arxiv.org/abs/2504.15027v1), 2025-04-21

(30) [LLMs as Data Annotators: How Close Are We to Human Performance](http://arxiv.org/abs/2504.15022v1), 2025-04-21

(31) [Stay Hungry, Stay Foolish: On the Extended Reading Articles Generation with LLMs](http://arxiv.org/abs/2504.15013v1), 2025-04-21

(32) [Efficient Pretraining Length Scaling](http://arxiv.org/abs/2504.14992v1), 2025-04-21

(33) [aiXamine: LLM Safety and Security Simplified](http://arxiv.org/abs/2504.14985v1), 2025-04-21

(34) [Evaluating LLMs on Chinese Topic Constructions: A Research Proposal Inspired by Tian et al. (2024)](http://arxiv.org/abs/2504.14969v1), 2025-04-21

(35) [SLO-Aware Scheduling for Large Language Model Inferences](http://arxiv.org/abs/2504.14966v1), 2025-04-21

(36) [Evaluating Code Generation of LLMs in Advanced Computer Science Problems](http://arxiv.org/abs/2504.14964v1), 2025-04-21

(37) [Efficient Document Retrieval with G-Retriever](http://arxiv.org/abs/2504.14955v1), 2025-04-21

(38) [Vector Embedding, Retrieval-Augmented Generation, CPU-NPU Collaboration, Heterogeneous Computing](http://arxiv.org/abs/2504.14941v1), 2025-04-21

(39) [EducationQ: Evaluating LLMs' Teaching Capabilities Through Multi-Agent Dialogue Framework](http://arxiv.org/abs/2504.14928v1), 2025-04-21

(40) [POLYRAG: Integrating Polyviews into Retrieval-Augmented Generation for Medical Applications](http://arxiv.org/abs/2504.14917v1), 2025-04-21

(41) [StableQuant: Layer Adaptive Post-Training Quantization for Speech Foundation Models](http://arxiv.org/abs/2504.14915v1), 2025-04-21

(42) [CRAVE: A Conflicting Reasoning Approach for Explainable Claim Verification Using LLMs](http://arxiv.org/abs/2504.14905v1), 2025-04-21

(43) [Hardware-based Heterogeneous Memory Management for Large Language Model Inference](http://arxiv.org/abs/2504.14893v1), 2025-04-21

(44) [Retrieval Augmented Generation Evaluation in the Era of Large Language Models: A Comprehensive Survey](http://arxiv.org/abs/2504.14891v1), 2025-04-21

(45) [Efficient Function Orchestration for Large Language Models](http://arxiv.org/abs/2504.14872v1), 2025-04-21

(46) [Natural Fingerprints of Large Language Models](http://arxiv.org/abs/2504.14871v1), 2025-04-21

(47) [OTC: Optimal Tool Calls via Reinforcement Learning](http://arxiv.org/abs/2504.14870v1), 2025-04-21

(48) [Transparentize the Internal and External Knowledge Utilization in LLMs with Trustworthy Citation](http://arxiv.org/abs/2504.14856v1), 2025-04-21

(49) [APIRAT: Integrating Multi-source API Knowledge for Enhanced Code Translation with LLMs](http://arxiv.org/abs/2504.14852v1), 2025-04-21

(50) [Language Models for Materials Discovery and Sustainability: Progress, Challenges, and Opportunities](http://arxiv.org/abs/2504.14849v1), 2025-04-21

(51) [Enhancing the Patent Matching Capability of Large Language Models via the Memory Graph](http://arxiv.org/abs/2504.14845v1), 2025-04-21

(52) [Establishing Reliability Metrics for Reward Models in Large Language Models](http://arxiv.org/abs/2504.14838v1), 2025-04-21

(53) [SQL-Factory: A Multi-Agent Framework for High-Quality and Large-Scale SQL Generation](http://arxiv.org/abs/2504.14837v1), 2025-04-21

(54) [Completing A Systematic Review in Hours instead of Months with Interactive AI Agents](http://arxiv.org/abs/2504.14822v1), 2025-04-21

(55) [DONOD: Robust and Generalizable Instruction Fine-Tuning for LLMs via Model-Intrinsic Dataset Pruning](http://arxiv.org/abs/2504.14810v1), 2025-04-21

(56) [On Self-improving Token Embeddings](http://arxiv.org/abs/2504.14808v1), 2025-04-21

(57) [Automatic Evaluation Metrics for Document-level Translation: Overview, Challenges and Trends](http://arxiv.org/abs/2504.14804v1), 2025-04-21

(58) [Automated Duplicate Bug Report Detection in Large Open Bug Repositories](http://arxiv.org/abs/2504.14797v1), 2025-04-21

(59) [gLLM: Global Balanced Pipeline Parallelism System for Distributed LLM Serving with Token Throttling](http://arxiv.org/abs/2504.14775v1), 2025-04-21

(60) [PLANET: A Collection of Benchmarks for Evaluating LLMs' Planning Capabilities](http://arxiv.org/abs/2504.14773v1), 2025-04-21

(61) [Knowledge Distillation and Dataset Distillation of Large Language Models: Emerging Trends, Challenges, and Future Directions](http://arxiv.org/abs/2504.14772v1), 2025-04-20

(62) [The Memorization Problem: Can We Trust LLMs' Economic Forecasts?](http://arxiv.org/abs/2504.14765v1), 2025-04-20

(63) [Steering Semantic Data Processing With DocWrangler](http://arxiv.org/abs/2504.14764v1), 2025-04-20

(64) [SWE-Synth: Synthesizing Verifiable Bug-Fix Data to Enable Large Language Models in Resolving Real-World Bugs](http://arxiv.org/abs/2504.14757v1), 2025-04-20

(65) [PROMPTEVALS: A Dataset of Assertions and Guardrails for Custom Production Large Language Model Pipelines](http://arxiv.org/abs/2504.14738v1), 2025-04-20

(66) [Pairwise or Pointwise? Evaluating Feedback Protocols for Bias in LLM-Based Evaluation](http://arxiv.org/abs/2504.14716v1), 2025-04-20

(67) [AI with Emotions: Exploring Emotional Expressions in Large Language Models](http://arxiv.org/abs/2504.14706v1), 2025-04-20

(68) [Video-MMLU: A Massive Multi-Discipline Lecture Understanding Benchmark](http://arxiv.org/abs/2504.14693v1), 2025-04-20

(69) [FarsEval-PKBETS: A new diverse benchmark for evaluating Persian large language models](http://arxiv.org/abs/2504.14690v1), 2025-04-20

(70) [Designing AI Systems that Augment Human Performed vs. Demonstrated Critical Thinking](http://arxiv.org/abs/2504.14689v1), 2025-04-20

(71) [An LLM-enabled Multi-Agent Autonomous Mechatronics Design Framework](http://arxiv.org/abs/2504.14681v1), 2025-04-20

(72) [Trans-Zero: Self-Play Incentivizes Large Language Models for Multilingual Translation Without Parallel Data](http://arxiv.org/abs/2504.14669v1), 2025-04-20

(73) [Efficient Federated Split Learning for Large Language Models over Communication Networks](http://arxiv.org/abs/2504.14667v1), 2025-04-20

(74) [Generative Multimodal Pretraining with Discrete Diffusion Timestep Tokens](http://arxiv.org/abs/2504.14666v1), 2025-04-20

(75) [A Case Study Exploring the Current Landscape of Synthetic Medical Record Generation with Commercial LLMs](http://arxiv.org/abs/2504.14657v1), 2025-04-20

(76) [LeetCodeDataset: A Temporal Dataset for Robust Evaluation and Efficient Training of Code LLMs](http://arxiv.org/abs/2504.14655v1), 2025-04-20

(77) [A Framework for Benchmarking and Aligning Task-Planning Safety in LLM-Based Embodied Agents](http://arxiv.org/abs/2504.14650v1), 2025-04-20

(78) [Relation-R1: Cognitive Chain-of-Thought Guided Reinforcement Learning for Unified Relational Comprehension](http://arxiv.org/abs/2504.14642v1), 2025-04-20

(79) [HLSTester: Efficient Testing of Behavioral Discrepancies with LLMs for High-Level Synthesis](http://arxiv.org/abs/2504.14641v1), 2025-04-20

(80) [Risk Assessment Framework for Code LLMs via Leveraging Internal States](http://dx.doi.org/10.1145/3696630.3728566), 2025-04-20

(81) [Harnessing Generative LLMs for Enhanced Financial Event Entity Extraction Performance](http://arxiv.org/abs/2504.14633v1), 2025-04-20

(82) [Towards Optimal Circuit Generation: Multi-Agent Collaboration Meets Collective Intelligence](http://arxiv.org/abs/2504.14625v1), 2025-04-20

(83) [A Hierarchical Framework for Measuring Scientific Paper Innovation via Large Language Models](http://arxiv.org/abs/2504.14620v1), 2025-04-20

(84) [Translation Analytics for Freelancers: I. Introduction, Data Preparation, Baseline Evaluations](http://arxiv.org/abs/2504.14619v1), 2025-04-20

(85) [UFO2: The Desktop AgentOS](http://arxiv.org/abs/2504.14603v1), 2025-04-20

(86) [a1: Steep Test-time Scaling Law via Environment Augmented Generation](http://arxiv.org/abs/2504.14597v1), 2025-04-20

(87) [HealthGenie: Empowering Users with Healthy Dietary Guidance through Knowledge Graph and Large Language Models](http://arxiv.org/abs/2504.14594v1), 2025-04-20

(88) [Phoenix: A Motion-based Self-Reflection Framework for Fine-grained Robotic Action Correction](http://arxiv.org/abs/2504.14588v1), 2025-04-20

(89) [Prompt-Hacking: The New p-Hacking?](http://arxiv.org/abs/2504.14571v1), 2025-04-20

(90) [NoWag: A Unified Framework for Shape Preserving Compression of Large Language Models](http://arxiv.org/abs/2504.14569v1), 2025-04-20

(91) [ReasoningV: Efficient Verilog Code Generation with Adaptive Hybrid Reasoning Model](http://arxiv.org/abs/2504.14560v1), 2025-04-20

(92) [Enhancing LLM-based Quantum Code Generation with Multi-Agent Optimization and Quantum Error Correction](http://arxiv.org/abs/2504.14557v1), 2025-04-20

(93) [LLM-Enabled In-Context Learning for Data Collection Scheduling in UAV-assisted Sensor Networks](http://arxiv.org/abs/2504.14556v1), 2025-04-20

(94) [REDEditing: Relationship-Driven Precise Backdoor Poisoning on Text-to-Image Diffusion Models](http://arxiv.org/abs/2504.14554v1), 2025-04-20

(95) [BookWorld: From Novels to Interactive Agent Societies for Creative Story Generation](http://arxiv.org/abs/2504.14538v1), 2025-04-20

(96) [Causality for Natural Language Processing](http://arxiv.org/abs/2504.14530v1), 2025-04-20

(97) [Are Vision LLMs Road-Ready? A Comprehensive Benchmark for Safety-Critical Driving Video Understanding](http://arxiv.org/abs/2504.14526v1), 2025-04-20

(98) [Biased by Design: Leveraging AI Biases to Enhance Critical Thinking of News Readers](http://arxiv.org/abs/2504.14522v1), 2025-04-20

(99) [Meta-Thinking in LLMs via Multi-Agent Reinforcement Learning: A Survey](http://arxiv.org/abs/2504.14520v1), 2025-04-20

(100) [SlimPipe: Memory-Thrifty and Efficient Pipeline Parallelism for Long-Context LLM Training](http://arxiv.org/abs/2504.14519v1), 2025-04-20
